src/ArrayObject.php

```diff
@@ -333,27 +333,27 @@ public function &offsetGet(mixed $key)
     /**
      * Sets the value at the specified key to value
      *
-     * @param TKey $key
+     * @param TKey $offset
      * @param TValue $value
      * @return void
      */
     #[ReturnTypeWillChange]
-    public function offsetSet(mixed $key, mixed $value)
+    public function offsetSet(mixed $offset, mixed $value)
     {
-        $this->storage[$key] = $value;
+        $this->storage[$offset] = $value;
```

Comment:
@Ocramius - Can we change parameter names to match inherited without considering it a BC break?

Reply:
IMO yes: we don't support named parameters